openedx/core/djangoapps/user_api/accounts/__init__.py

```diff
@@ -40,83 +40,83 @@
-USERNAME_BAD_TYPE_MSG = "Username must be a string."
-EMAIL_BAD_TYPE_MSG = "Email must be a string."
-PASSWORD_BAD_TYPE_MSG = "Password must be a string."
+USERNAME_BAD_TYPE_MSG = "Username must be a string"
+EMAIL_BAD_TYPE_MSG = "Email must be a string"
+PASSWORD_BAD_TYPE_MSG = "Password must be a string"
 
 # Translators: These messages are shown to users who do not enter information
 # into the required field or enter it incorrectly.
-REQUIRED_FIELD_NAME_MSG = _("Enter your full name.")
-REQUIRED_FIELD_FIRST_NAME_MSG = _("Enter your first name.")
-REQUIRED_FIELD_LAST_NAME_MSG = _("Enter your last name.")
-REQUIRED_FIELD_CONFIRM_EMAIL_MSG = _("The email addresses do not match.")
+REQUIRED_FIELD_NAME_MSG = _("Enter your full name")
+REQUIRED_FIELD_FIRST_NAME_MSG = _("Enter your first name")
+REQUIRED_FIELD_LAST_NAME_MSG = _("Enter your last name")
+REQUIRED_FIELD_CONFIRM_EMAIL_MSG = _("The email addresses do not match")
 REQUIRED_FIELD_CONFIRM_EMAIL_TEXT_MSG = _("Enter your confirm email")
-REQUIRED_FIELD_COUNTRY_MSG = _("Select your country or region of residence.")
-REQUIRED_FIELD_PROFESSION_SELECT_MSG = _("Select your profession.")
-REQUIRED_FIELD_SPECIALTY_SELECT_MSG = _("Select your specialty.")
-REQUIRED_FIELD_PROFESSION_TEXT_MSG = _("Enter your profession.")
-REQUIRED_FIELD_SPECIALTY_TEXT_MSG = _("Enter your specialty.")
-REQUIRED_FIELD_STATE_MSG = _("Enter your state.")
-REQUIRED_FIELD_CITY_MSG = _("Enter your city.")
-REQUIRED_FIELD_GOALS_MSG = _("Tell us your goals.")
-REQUIRED_FIELD_LEVEL_OF_EDUCATION_MSG = _("Select the highest level of education you have completed.")
+REQUIRED_FIELD_COUNTRY_MSG = _("Select your country or region of residence")
+REQUIRED_FIELD_PROFESSION_SELECT_MSG = _("Select your profession")
+REQUIRED_FIELD_SPECIALTY_SELECT_MSG = _("Select your specialty")
+REQUIRED_FIELD_PROFESSION_TEXT_MSG = _("Enter your profession")
+REQUIRED_FIELD_SPECIALTY_TEXT_MSG = _("Enter your specialty")
+REQUIRED_FIELD_STATE_MSG = _("Enter your state")
+REQUIRED_FIELD_CITY_MSG = _("Enter your city")
+REQUIRED_FIELD_GOALS_MSG = _("Tell us your goals")
+REQUIRED_FIELD_LEVEL_OF_EDUCATION_MSG = _("Select the highest level of education you have completed")
 REQUIRED_FIELD_YEAR_OF_BIRTH_MSG = _("Select your year of birth")
 REQUIRED_FIELD_GENDER_MSG = _("Select your gender")
-REQUIRED_FIELD_MAILING_ADDRESS_MSG = _("Enter your mailing address.")
+REQUIRED_FIELD_MAILING_ADDRESS_MSG = _("Enter your mailing address")
 
 # HIBP Strings
 AUTHN_LOGIN_BLOCK_HIBP_POLICY_MSG = _(
-    'Our system detected that your password is vulnerable. Change your password so that your account stays secure.'
+    'Our system detected that your password is vulnerable. Change your password so that your account stays secure'
 )
 AUTHN_LOGIN_NUDGE_HIBP_POLICY_MSG = _(
-    'Our system detected that your password is vulnerable. '
-    'We recommend you change it so that your account stays secure.'
+    'Our system detected that your password is vulnerable '
```

Comment:
I guess we don't need to remove the dot for the first sentence of this message. Or maybe needed to convert it into a comma, can you please confirm?
`Our system detected that your password is vulnerable, we recommend you change it so that your account stays secure`